shared/common-adapters/input.native.js

```diff
@@ -248,16 +223,15 @@ class Input extends Component<Props, State> {
       onBlur: this._onBlur,
       onChangeText: this._onChangeText,
       onFocus: this._onFocus,
-      onKeyDown: this._onKeyDown,
```

Comment:
the right event is `onKeyPress`. As an aside, that means we can never trigger that Android emoji crasher with our workaround.